modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -318,15 +333,17 @@ def drop_network(self, nodes=None):
         dsc_ports = str(dsc_spi.port) if not hasattr(dsc_spi, 'port_range') or dsc_spi.port_range < 1 else str(
             dsc_spi.port) + ':' + str(dsc_spi.port + dsc_spi.port_range)
 
-        cmd = f"sudo iptables -I %s 1 -p tcp -m multiport --dport {dsc_ports},{cm_ports} -j DROP"
+        if net_part in (IgniteAwareService.NetPart.ALL, IgniteAwareService.NetPart.INCOMING):
+            node.account.ssh_client.exec_command(
+                f"sudo iptables -I INPUT 1 -p tcp -m multiport --dport {dsc_ports},{cm_ports} -j DROP")
+            node.account.ssh_client.exec_command(
+                f"sudo iptables -I FORWARD 1 -p tcp -m multiport --dport {dsc_ports},{cm_ports} -j DROP")
```

Comment:
Why forward is incoming while it's outcoming as well?

Reply:
Fixed